src/frontend/src/css/vendors.less

```diff
@@ -12,4 +11,6 @@
 @import "../../node_modules/nav-frontend-spinner-style/src/spinner-style";
 @import "../../node_modules/nav-frontend-skjema-style/src/index";
 @import "../../node_modules/nav-frontend-modal-style/src/index";
+@import "../../node_modules/nav-frontend-tabs-style/src/tabs-style";
```

Comment:
Med CRACO mener jeg du skal kunne skrive import  "~/nav-frontend-xxxx/" etc. (med ~ tegnet i stedet for "../../xxxx")